samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/delivery/SectionContentsBean.java

```diff
@@ -97,7 +97,7 @@ public class SectionContentsBean extends SpringBeanAutowiringSupport implements
   private Integer sectionAuthorType;
   private Integer questionOrdering;
   private Integer numberToBeDrawn;
-  private Long poolIdToBeDrawn;
```

Comment:
Why are you changing the access level to protected? The field seems to only be accessed through the public setter.

Reply:
This comes from a previous version, I tried to access it directly but changed my mind.